ci/validate.sh

```diff
@@ -52,17 +52,20 @@ if [[ "$CIRRUS_PR_TITLE" =~ CI:DOCS ]]; then
   exit 0
 fi
 
-# Variable is defined by Cirrus-CI at runtime
-# shellcheck disable=SC2154
-if ! git diff --name-only ${CIRRUS_BASE_SHA}..HEAD | grep -q IMG_SFX; then
+# Fix "Not a valid object name main" error from Cirrus's
+# incomplete checkout.
+git remote update origin
+# Determine where PR branched off of $CIRRUS_BASE_BRANCH
+base_sha=$(git merge-base origin/${CIRRUS_BASE_BRANCH:-main} HEAD)
```

Comment:
I remember trying `origin/` in my podman PR, and it failed so I removed it. I can't remember what the failure was. Do you remember why you added `origin/` ?

Reply:
Yes, it's in the comment above the `git remote update origin` :wink: